src/ik.rs

```diff
@@ -109,12 +94,12 @@ impl Default for Constraints {
     /// ```
     fn default() -> Self {
         Self {
-            position_x: default_position_x(),
-            position_y: default_position_y(),
-            position_z: default_position_z(),
-            rotation_x: default_rotation_x(),
-            rotation_y: default_rotation_y(),
-            rotation_z: default_rotation_z(),
+            position_x: default_true(),
+            position_y: default_true(),
+            position_z: default_true(),
+            rotation_x: default_true(),
+            rotation_y: default_true(),
+            rotation_z: default_true(),
```

Comment:
I thought that this can be `position_x: true`, but `default_true()` is the same as the serde default actually.. uhhm.